### ISSUE TITLE
ios emulator stays white even though building successfully (polymer-app)

### ISSUE THREAD

**Issue by sLatzko** [MobileChromeApps/mobile-chrome-apps#563]
Some weeks ago I opened an issue #546 where I couldn't build my app on ios even though is was running on android without any problems.

I've updated ```cca``` and tried again. The errors were gone - the **build succeeds** - but the emulator screen **stayed white**. 

Only xCode shows 4 warnings. 
```
/Users/Stefan/joh-dpa/platforms/ios/Johanniter Dienstplan/Plugins/org.chromium.storage/ChromeStorage.m:184:91: Implicit conversion loses integer precision: 'NSUInteger' (aka 'unsigned long') to 'int' 
```
```
/Users/Stefan/joh-dpa/platforms/ios/Johanniter Dienstplan/Plugins/org.apache.cordova.statusbar/CDVStatusBar.m:182:40: Implicit conversion from enumeration type 'UIInterfaceOrientation' (aka 'enum UIInterfaceOrientation') to different enumeration type 'UIDeviceOrientation' (aka 'enum UIDeviceOrientation')
/Users/Stefan/joh-dpa/platforms/ios/Johanniter Dienstplan/Plugins/org.apache.cordova.statusbar/CDVStatusBar.m:204:44: Implicit conversion from enumeration type 'UIInterfaceOrientation' (aka 'enum UIInterfaceOrientation') to different enumeration type 'UIDeviceOrientation' (aka 'enum UIDeviceOrientation')
/Users/Stefan/joh-dpa/platforms/ios/Johanniter Dienstplan/Plugins/org.apache.cordova.statusbar/CDVStatusBar.m:373:52: Implicit conversion from enumeration type 'UIInterfaceOrientation' (aka 'enum UIInterfaceOrientation') to different enumeration type 'UIDeviceOrientation' (aka 'enum UIDeviceOrientation')
```

xCode Console Output:
```

2015-04-29 20:49:26.631 Johanniter Dienstplan[2689:71117] [CDVTimer][file] 20.545006ms
2015-04-29 20:49:26.900 Johanniter Dienstplan[2689:71117] [CDVTimer][statusbar] 269.464016ms
2015-04-29 20:49:26.901 Johanniter Dienstplan[2689:71117] [CDVTimer][chromebootstrap] 0.396013ms
2015-04-29 20:49:26.901 Johanniter Dienstplan[2689:71117] [CDVTimer][chromeextensionurls] 0.063002ms
2015-04-29 20:49:26.901 Johanniter Dienstplan[2689:71117] [CDVTimer][chromenavigation] 0.037014ms
2015-04-29 20:49:26.901 Johanniter Dienstplan[2689:71117] [CDVTimer][TotalPluginStartup] 291.731000ms
2015-04-29 20:49:27.270 Johanniter Dienstplan[2689:71117] Resetting plugins due to page load.
2015-04-29 20:49:28.011 Johanniter Dienstplan[2689:71117] Finished load of: chrome-extension://hnnhphheambmlpoeopfoggbohahhlccm/build.html
```

I found a similar issue #450. I already run my polymer app with vulcanize, so this shouldn't be a problem, right? Is there anything I can do to solve this?

**Comment by agrieve** [comment]
Best thing to try is attach remote web inspector via safari, then hit command-R to refresh the page and see what errors spit out on the console.

**Comment by sLatzko** [comment]
You helped me a lot again. Thank you. 

Turns out I've gotten this 
```ReferenceError: Can't find variable: query```
error message. It is caused by a `?` in a `href="mailto:someone@some.where?subject=Feedback>`
https://github.com/Polymer/polymer/issues/1347

I commented it out and this particular error vanished. Just to find out that I now get following error:
```TypeError: Attempted to assign to readonly property.```
As far as I have understood, it is a Safari Bug related to `use strict`.
http://stackoverflow.com/questions/26044056/typeerror-attempted-to-assign-to-readonly-property-in-angularjs-application-on
https://github.com/Polymer/polymer/issues/1347
